go/vt/vttablet/tabletserver/vstreamer/vstreamer_test.go

```diff
@@ -1212,9 +1212,6 @@ func expectLog(ctx context.Context, t *testing.T, input interface{}, ch <-chan [
 					t.Fatalf("%v (%d): event: %v, want commit", input, i, evs[i])
 				}
 			default:
-				if evs[i].Timestamp == 0 {
```

Comment:
Is timestamp not testable any more?

Reply:
Until this change, all events were reporting their timestamp from the binlog. But this new OTHER event is invented by us, and we cannot give it a reasonable timestamp. Giving it a "now" value is risky because it may disagree with the next event we'll see from the binlog.

It turns out vreplication already checks for the 0-value. I've updated the protobuf documentation to indicate that it can now be 0.